src/jitter.rs

```diff
@@ -103,6 +119,20 @@ impl fmt::Debug for JitterRng {
     }
 }
 
+impl Clone for JitterRng {
+    fn clone(&self) -> JitterRng {
+        JitterRng {
+            data: self.data,
+            rounds: self.rounds,
+            timer: self.timer,
+            mem_prev_index: self.mem_prev_index,
```

Comment:
Do you think `mem_prev_index` should be adjusted?

Reply:
I think we are okay leaving it as is. They probably have different locations for the memory slice on the stack. Having the same location twice should not make the CPU much better at predicting. And after the first round the values will diverge, as `memaccess` is called a variable number of times.